Removed commented out line for ignoring coding standards

dverkade opened the pull request:
Removed commented out line for ignoring coding standards, this line should not be in the code and is removed for cleaner more readable code.

dverkade committed:
Removed commented out line for ignoring coding standards

Removed commented out line for ignoring coding standards, this line should not be in the code and is removed for cleaner more readable code.

orlangur commented:
There is [more than one](https://github.com/magento/magento2/search?q=codingStandardsIgnoreFile) occurrence of such annotation in source code currently.

At some point a huge mistake was made and instead of automatic fix of PSR-2 violations all problematic files were just excluded from coding style check. To fix this `@codingStandardsIgnoreFile` annotation usage must be prohibited and all existing violations corrected. Until then coding style violations elimination here and there seems to be a Sisyphean task to me.

You can see that after you restored coding style check, static test started to fail: https://travis-ci.org/magento/magento2/jobs/165630323 This error is easy to fix but not automatically.

dverkade committed:
Changed line length

Changed line length so that the @ignoreCodingStandard which was removed will not throw an exception anymore.

dverkade committed:
Update Factory.php

vrann commented:
@dverkade Appreciate your participation in the Magento community!
